classes/local/table/process_errors_table.php

```diff
@@ -165,6 +165,27 @@ protected function show_hide_link($column, $index) {
         return '';
     }
 
+    /**
+     * Show custom nothing to display message.
+     * @return void
+     */
+    public function print_nothing_to_display() {
+        global $OUTPUT;
+
+        // Render the dynamic table header.
+        echo $this->get_dynamic_table_html_start();
+
+        // Render button to allow user to reset table preferences.
+        echo $this->render_reset_button();
```

Comment:
We have already asked Aaron about this. Do we really need these buttons for an empty table? We have seen that in multiple plugins but why is it necessary?

Reply:
I don't really know, I just copied the print_nothing_to_display() function and edited the string

Reply:
Wird schon seine Gründe haben